### PR TITLE
Add Dashboard icon and deleted dropdown of the skills

### LINKED ISSUE #2505: Add Dashboard icon
Add a Dashboard (linking to /dashboard) icon between the name dropdown and the right support icon.

Also delete the "skills" dropdown menu item from the top menu.

 Screenshot from 2019-07-10 13-34-13

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2505

Changes: 
- added a Dashboard (linking to /dashboard) icon
- deleted the "skills" dropdown menu

Demo Link: https://pr-2521-fossasia-susi-web-chat.surge.sh

Screenshots of the change:
Capture
Capture

### CODE REVIEW

**Comment by samagragupta** [comment]
@akshatnitd Fixed

**Comment by mariobehling** [comment]
Please check and fix Travis.

File: src/components/NavigationBar/index.js [hunk 1]
```diff
@@ -249,31 +250,41 @@ class NavigationBar extends Component {
...
-                          name="User Avatar"
-                          src={userAvatar}
-                          size="32"
-                        />
-                        <UserDetail>{!userName ? email : userName}</UserDetail>
-                        <ExpandMore />
-                      </FlexContainer>
-                    </div>
-                  </StyledIconButton>
+                  <div>
```
Comment (akshatnitd) [inline src/components/NavigationBar/index.js]: Is this extra nesting required?

Comment (samagragupta) [reply src/components/NavigationBar/index.js]: It should be in a single wrapper.

Comment (samagragupta) [reply src/components/NavigationBar/index.js]: Removing that it says:
Syntax error: Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>? (281:21)

File: src/components/NavigationBar/index.js [hunk 2]
```diff
@@ -249,31 +250,41 @@ class NavigationBar extends Component {
...
+                        </FlexContainer>
+                      </div>
+                    </StyledIconButton>
+                    <IconButton
+                      color="inherit"
+                      onClick={() => history.push('/dashboard')}
+                    >
+                      <Dashboard />
+                    </IconButton>
+                  </div>
```
Comment (fragm3) [inline src/components/NavigationBar/index.js]: Use React.fragment

Comment (samagragupta) [reply src/components/NavigationBar/index.js]: @fragm Done

**Comment by fragm3** [comment]
@mariobehling The skills dropdown is actually useful for create skill, create bot shortcut. Not sure why it needs to be deleted

**Comment by mariobehling** [comment]
@fragm3 We need to make space for a search bar for skills and will add a "create" button to the right of the bar later.

**Pull request merged**